src/xdocs/index.xml.vm

```diff
@@ -69,7 +69,8 @@
         If you want to live on the bleeding edge, you can <a
         href="https://github.com/checkstyle/checkstyle">checkout</a>
         the current development code from GitHub and compile
-        yourself.
+        yourself. Or use <a href="https://jitpack.io/">jitpack.io</a> to build
+        and get arifacts you need (you can build even your forked repo).
```

Comment:
`you can build even your` => `you can even build your`

Reply:
fixed.